### PR TITLE
Improve Travis CI configuration

### PR DESCRIPTION
**Pull request by szepeviktor**
No more conditions :)
Speed up by using cache.

### CODE REVIEW

**Comment by szepeviktor** [comment]
All 🍏 
https://travis-ci.com/github/szepeviktor/php-annotations/builds/213928025

**Comment by szepeviktor** [comment]
Please consider migrating to travis-ci.COM

File: .travis.yml [hunk 1]
```diff
@@ -1,34 +1,43 @@
+# TravisCI configuration for php-annotations/php-annotations
+
 language: php
+os:
+  - "linux"
+dist: "xenial"
+
+php:
```
Comment (aik099) [inline .travis.yml:8]: 1. why you've rearranged the sections
2. why you've added double quotes around arguments (everything worked without this before)

Depending on the answers I might ask you to revert the above-mentioned changes.

**Review by aik099 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (szepeviktor) [reply .travis.yml:8]: Hello!

1. now it is in chronological order, as it happens during execution
2. if you always quote YAML strings you don't have to decide one-by-one whether it is necessary (e.g. floats like 7.4)

Comment (szepeviktor) [reply .travis.yml:8]: actually all string should be quoted but I wasn't daring...

File: .travis.yml [hunk 2]
```diff
@@ -1,34 +1,43 @@
...
   - composer install --prefer-source
 
-script: php test/test.php
+script:
+  - php test/test.php
 
 after_script:
-  - if [[ $COVERAGE != no ]]; then wget https://scrutinizer-ci.com/ocular.phar && php ocular.phar code-coverage:upload --format=php-clover clover.xml; fi
+  - wget https://scrutinizer-ci.com/ocular.phar
+  - php ocular.phar code-coverage:upload --format=php-clover clover.xml
```
Comment (aik099) [inline .travis.yml:43]: Keep these changes.

File: .travis.yml (hunk 1 above)
Comment (aik099) [reply .travis.yml:8]: Massive changes are best to discuss before doing them.

IMO adding double quotes all over the place and rearranging stuff without asking if that is ever needed is both non-productive from the developer and a code-reviewer side.

Please try to revert above mentioned changes and we'll see if what's left can be merged.

**Review by aik099 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (szepeviktor) [reply .travis.yml:8]: Please see #134

**Pull request closed without merging**